widgets/nuxeo-user-suggestion.html

```diff
@@ -278,3 +286,81 @@
     }
   </script>
 </dom-module>
+
+<dom-module id="nuxeo-user-group-formatter">
```

Comment:
This seems like a useful element. Perhaps we could consider moving it into a dedicated file?

Reply:
:+1:

Reply:
agreed